app/middlewares/userValidations.js

```diff
@@ -57,3 +57,34 @@ exports.verifyAuthentication = (req, res, next) => {
     res.status(401).send('You are not logged in');
   }
 };
+
+exports.checkAdmin = (req, res, next) => {
+  if (req.headers.authorization) {
+    const tokenString = req.headers.authorization.replace('Bearer ', '');
+    const token = jwt.decode(tokenString, config.common.session.secret);
+    User.findOne({
+      where: { email: token.email }
+    }).then(anUser => {
+      if (anUser && token) {
```

Comment:
i think it would be better here to do a single if, something like `if(token && anUser && anUser.admin){}`, and in the `else{}` simply return a generic 401 error.